docs/specs/openapi.yml

```diff
@@ -110,3 +122,17 @@ definitions:
       message:
         type: string
         description: A human readable error message.
+  Person:
+    type: object
+    properties:
+      id:
+        type: integer
+        format: int64
+        description: Unique ID for a person
+      first_name:
```

Comment:
In my opinion, there's no need in having both first and last names. Since it's an example, let's keep things simple and dumb. Here I mean to have only one `name` attribute instead of separate first and last one. What do you think? Would you mind to change? :)

(I'm sorry for being nasty, just like when everything pleases my eyes.)